internal/codecs/datetime_test.go

```diff
@@ -0,0 +1,56 @@
+// This source file is part of the EdgeDB open source project.
+//
+// Copyright 2020-present EdgeDB Inc. and the EdgeDB authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codecs
+
+import (
+	"testing"
+	"unsafe"
+
+	"github.com/edgedb/edgedb-go/internal/buff"
+	types "github.com/edgedb/edgedb-go/internal/edgedbtypes"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLocalDateTimeDecode(t *testing.T) {
+	r := buff.SimpleReader([]byte{
+		0x00, 0x02, 0x2b, 0x35, 0x9b, 0xc4, 0x10, 0x00,
+	})
+
+	var result types.LocalDateTime
```

Comment:
For edgedb-go users it will be `edgedb.LocalDateTime`, right?

Reply:
correct